content/en/blog/2023/otel-in-focus-05.md

```diff
@@ -11,7 +11,7 @@ the sky is blue, and it's time to run down the latest updates from the
 OpenTelemetry project!
 
 Are you a maintainer with something you’d like featured here? Get in touch with
-me [via email](mailto:austin+otel@ap2.io), or on the
+[Austin Parker via email](mailto:austin+otel@ap2.io), or on the
```

Comment:
This doesn't seem correct ... I think the original intention was "get in touch with me via X or Y" rather than "get in touch with X or Y" ...

Reply:
The goal of this change is to support screen readers that read out lists of links -- they omit any context, so whatever is meaningful about a link needs to be _in_ the link text.

One could make the entire phrase a link as in:
> [Get in touch with Austin Parker via email](mailto:austin+otel@ap2.io)

But that felt a bit overkill (it isn't absolutely wrong, and I'm willing to do it, but it's also a bit much).